src/main/java/net/digimonworld/decode/randomizer/settings/PatchSettings.java

```diff
@@ -1049,7 +1066,7 @@ public Map<String, Object> serialize() {
         map.put("patchDisable90FBattles", patchDisable90FBattles.get());
         map.put("patchMovementSpeed", patchMovementSpeed.get());
         map.put("patchAddRecolorDigimon", patchAddRecolorDigimon.get());
-        
+        map.put("patchEvoListOrder", patchEvoListOrder.get());	
```

Comment:
```suggestion
        map.put("patchEvoListOrder", patchEvoListOrder.get());
        
```

Extra Leerzeile zwischen map.put und return.